Makefile

```diff
@@ -4,7 +4,7 @@ IMG ?= controller:latest
 # Controller-gen tool
 CONTROLLER_GEN ?= go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=true,crdVersions=v1"
```

Comment:
Unfortunately, we need to keep this consistent with what is in the machine-api-operator for the time being. https://github.com/openshift/machine-api-operator/blob/master/install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml#L1 shows that is still using v1beta1.

I think that means deferring the fix for this until later in the 4.7 cycle, when we have completed the work to remove the metal3 management logic from the MAO. Unless we also need to backport the fix to 4.6, in which case I would expect a patch in the MAO repository to land before we make the change here.

Reply:
I think the point here is to copy this into the MAO, since the explain output is currently broken in 4.6.

Reply:
I believe it will be safe to copy what is currently generated in this PR over to MAO within the 4.6 timeline, can we can a PR raised for that and attached to the same bug?

Reply:
This generated CRD is different from the one in MAO specifically around https://github.com/openshift/machine-api-operator/blob/master/install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml#L98-L101. Either we have to update our code here to generate the enum or make the changes in MAO independent of these changes.

Reply:
The enum you've highlighted appears to be present in the version in this PR as well? 
https://github.com/openshift/cluster-baremetal-operator/blob/2c5c0e6e9e6d8a117a802e0b9d4307af71142df5/config/crd/bases/metal3.io_provisionings.yaml#L48-L51

Reply:
Yes, https://github.com/openshift/cluster-baremetal-operator/pull/19 took care of that.

Reply:
Ok, so it should be safe to copy this generated file over to MAO and make sure they're synced?